.travis.yml

```diff
@@ -23,16 +23,18 @@ matrix:
   include:
   - python: 2.7
     env: TEST_TARGET=default
-  - python: 3.6
+  - python: 3.7
```

Comment:
Since this is not really testing anything let's bump it to be "future proof" once the test is available.